feat(Progress): add new prop orientation to display vertically (#18909)
ant-design/ant-design: components/progress/demo/orientation.md

---

> order: 0

Reviewer: ```suggestion
order: 11
```

title:

>   zh-CN: 取向

Reviewer 2: 方向

  en-US: Orientation
---

## zh-CN

> 通过设置`orientaton =“ vertical”`，可以将方向从水平更改为垂直。

Reviewer 2: ```
通过设置 `orientaton="vertical"`，可以将方向从水平更改为垂直。
```

Reviewer: ```suggestion
通过设置 `orientation="vertical"`，可以将方向从水平更改为垂直。
```

Author: Done

## en-US

> By setting `orientaton="vertical"`, you can change the orientantion from horizontal to vertical.

Reviewer: ```suggestion
By setting `orientation="vertical"`, you can change the orientnation from horizontal to vertical.
```

---
order: 11
title:
  zh-CN: 方向
  en-US: Orientation
---

## zh-CN

> 通过设置`orientaton=“vertical”`，可以将方向从水平更改为垂直。

Reviewer 3: ```diff
- 通过设置`orientaton=“vertical”`，
+ 通过设置 `orientaton="vertical"`，
```

Reviewer 4: @Larissagilliane @afc163 Progress 没有找到 orientation 这个属性呢
